Readme.md

```diff
@@ -37,6 +37,26 @@ items = [1,2,3]
 Parallel.each( -> { items.pop || Parallel::Stop }) { |number| ... }
 ```
 
+Reduce will return results colleced from each worker. So you need to merge them on the end.
+```Ruby
+result = Parallel.reduce(['a','b','c','d','a','b','c','d']) do |result,x|
+  result ||= Set.new
+  result << x
+  result
+end
+result.compact.reduce(&:+)
```

Comment:
why does it need compact ?
... maybe do `'x' + 'p'` to show that processing was done

Reply:
That is fixed now we do not need it.